src/Components/App/App.js

```diff
@@ -14,7 +14,7 @@ const App = () => {
   const [userLogs, setUserLogs] = useState(entryData);
   const [feeling, setFeeling] = useState('');
   const [activity, setActivity] = useState('');
-  const [journal, setJournal] = useState('');
+  const [journal, setJournal] = useState(entryData[0]);
```

Comment:
I'm wondering if it would be _better_ to start with the state empty and use `useEffect` to bring in the data (to mock our network request)?

Reply:
+1 Amber I think that might be best practice but Im not 100% sure